### ISSUE TITLE
Creating a more visible banner for old process documents

### ISSUE THREAD

**Issue by egekorkan** [w3c/w3process#855]
If I click to a link pointing to a relative link within a process document (e.g. https://www.w3.org/2021/Process-20211102/#registries), my browser automatically goes there as expected. However, I miss the little warning at the top of the document, see below:

![image](https://github.com/w3c/w3process/assets/20195407/fb5e2dc3-7595-4eb8-8a38-1c229141bb8d)

Looking elsewhere: clicking on a proposed rec's relative link (e.g. https://www.w3.org/TR/2020/PR-wot-thing-description-20200130/#dataschema) makes it much more obvious that there is a newer version.

![image](https://github.com/w3c/w3process/assets/20195407/7598745e-56c5-47b9-8ef7-dece18fadaf8)

Is the process document's implementation intentional?

**Comment by frivoal** [comment]
@plehegar I agree with the request. This is probably something that should be moved to the Team, as these warnings are injected by the server system, rather than built into the source of the process.

**Comment by plehegar** [comment]
I'm not sure where to transfer this issue but it's now on my todo list
